packages/dataviews/src/constants.ts

```diff
@@ -58,6 +58,10 @@ export const sortLabels = {
 	asc: __( 'Sort ascending' ),
 	desc: __( 'Sort descending' ),
 };
+export const sortLabelsShort = {
+	asc: __( 'ASC' ),
```

Comment:
AFAIK we rarely capitalize strings; instead, capitalization is achieved cosmetically with CSS.

Also, note that this can be problematic for translation in some languages. If we can, I'd recommend we use the full words (ascending / descending)

Reply:
@jorgefilipecosta thanks for the screenshot. This is exactly what will happen with the translations, if folks start translating and there's no good abbreviation in their language.

I wonder if a worthy alternative is to use icons instead of ASC/DESC text labels. We have a few up/down arrow icons available that we could use here. @jameskoster WDYT?

Reply:
Yup, let's try an icon :)